packages/reason-css-parser/lib/Parser.re

```diff
@@ -1599,7 +1599,7 @@ and property_text_emphasis = [%value.rec
 ]
 and property_text_emphasis_color = [%value.rec "<color>"]
 and property_text_emphasis_position = [%value.rec
-  "[ 'over' | 'under' ] && [ 'right' | 'left' ]"
+  "[ 'over' | 'under' ] && [ 'right' | 'left' ]?"
```

Comment:
The change on the spec is necessary? Can you add a comment on why?

Reply:
Yes, it is necessary because it is part of the formal syntax -- https://drafts.csswg.org/css-text-decor/#text-emphasis-position-property